Marsiple/Common/Extensions/UIAlertController-Marsiple.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UIAlertController-Marsiple.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 27/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import UIKit
+
+extension UIAlertController {
+    static func alertStyle(title: String, message: String, cancelActionTitle: String, cancelActionHandler: ((UIAlertAction) -> Void)?) -> UIAlertController {
```

Comment:
Hmm, `alertStyle` as the `method` name? Not sure that I understand, as the name doesn't provide any meaningful context to how and where it's used.

Reply:
@matej494, alright, can work with that.